### ISSUE TITLE
MSI Installer - Installation Path

### ISSUE THREAD

**Issue by cwchristerw** [yggdrasil-network/yggdrasil-go#657]
I would want to install yggdrasil to C:\Program Files\Yggdrasil Network current it goes to C:\Program Files\Yggdrasil it cannot be changed.

**Comment by Vort** [comment]
For now you can move files manually and change service command line with regedit:
`HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\services\yggdrasil`, `ImagePath`

**Comment by cwchristerw** [comment]
Okay you're correct, but I think there should be option to set these during installation process.

**Comment by Vort** [comment]
Yes, you are right. Another thing to think about - is where to put config file and logs if default path is changed. If user want to move exe, he may also want to move other files as well.

**Comment by cwchristerw** [comment]
You're correct in previous versions I have stored configuration in same folder as exe file :smile:
